tensorrt_llm/_torch/autotuner.py

```
@@ -1244,29 +1244,57 @@ def _profile_runners(
+                    input_tensors, profile, **kwargs)
+                total_pairs += len(all_valid_tactics)
+                candidates.append(
+                    (runner_id, runner, runner_arg_names, all_valid_tactics))
+        except Exception as e:
+            # Same reason as the tactic-profiling handler below (#13469): clear
+            # any pending CUDA error here, or it surfaces later in an unrelated
+            # call.
+            try:
+                torch.cuda.synchronize()
+            except Exception:
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)